solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -128,6 +128,17 @@ contract RandomBeacon is Ownable {
     ///         `maliciousDkgResultSlashingAmount`.
     uint256 public maliciousDkgResultSlashingAmount;
 
+    /// @notice Duration of the sortition pool rewards ban imposed on operators
+    ///         who missed their turn for relay entry submission.
```

Comment:
Not only relay entry submission, @tomaszslabon will also use it for DKG result submission. Hence, I'd suggest s/relay entry submission/relay entry or DKG result submission.

Reply:
See https://github.com/keep-network/keep-core/pull/2707/commits/6be521c1ffbd85c473e05aee899ab786382f4eba